src/main/java/com/demo/classroom/ClassroomApplication.java

```diff
@@ -3,10 +3,19 @@
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
+import io.github.cdimascio.dotenv.Dotenv;
+
 @SpringBootApplication
 public class ClassroomApplication {
 
 	public static void main(String[] args) {
+		
+		Dotenv dotenv = Dotenv.load();
+        System.setProperty("spring.datasource.url", dotenv.get("SPRING_DATASOURCE_URL"));
+        System.setProperty("spring.datasource.username", dotenv.get("SPRING_DATASOURCE_USERNAME"));
+        System.setProperty("spring.datasource.password", dotenv.get("SPRING_DATASOURCE_PASSWORD"));
```

Comment:
Is there any way to directly assign this values to yml?

Reply:
Done.